crates/revm_precompiles/src/lib.rs

```diff
@@ -112,6 +112,7 @@ impl Precompiles {
         static INSTANCE: OnceCell<Precompiles> = OnceCell::new();
         INSTANCE.get_or_init(|| {
             let fun = vec![
+                #[cfg(any(feature = "k256_ecrecover", feature = "secp256k1"))]
```

Comment:
I dont want to disable ECRECOVER if default is not present, it seems like a footgun. As I said here I am fine with build failure: https://github.com/bluealloy/revm/pull/252#issuecomment-1301929069

Having this would make this more user-friendly: https://doc.rust-lang.org/std/macro.compile_error.html

Something like
```
#[cfg(all(not(feature = "k256_ecrecover"), not(feature = "secp256k1")))]
compile_error!(
    "To support ecrecover precompile please enable one of these two features, `k256_ecrecover` or `secp256k1`
);
```

Reply:
Added the above mentioned `compile_error` to the `secp256k1` module